frontend/src/metabase/components/LeaveConfirmationModal/LeaveConfirmationModalContent.tsx

```diff
@@ -0,0 +1,26 @@
+import { t } from "ttag";
+
+import ConfirmContent from "metabase/components/ConfirmContent";
+
+interface Props {
+  onAction?: () => void;
+  onCancel?: () => void;
+  onClose?: () => void;
+}
+
+export const LeaveConfirmationModalContent = ({
+  onAction,
+  onCancel,
+  onClose,
+}: Props) => (
+  <ConfirmContent
+    cancelButtonText={t`Cancel`}
+    confirmButtonText={t`Leave anyway`}
+    data-testid="leave-confirmation"
```

Comment:
I wanted to keep the `data-testid` short to increase the probability of assertions to fit 1 LOC.